UI/ChatUI/ChatUI/ChatUI/MainPage.xaml

```diff
@@ -0,0 +1,427 @@
+										Content="{TemplateBinding Content}"
+										ContentTemplate="{TemplateBinding ContentTemplate}"
+										ContentTransitions="{TemplateBinding ContentTransitions}" />
+								</Grid>
+								<!--
+									The 'Xg' text simulates the amount of space one line of text will occupy.
+									In the DataPlaceholder state, the Content is not loaded yet so we
+									approximate the size of the item using placeholder text.
+								-->
+								<TextBlock
+									x:Name="PlaceholderTextBlock"
+									Margin="{TemplateBinding Padding}"
+									AutomationProperties.AccessibilityView="Raw"
+									Foreground="{x:Null}"
+									IsHitTestVisible="False"
+									Opacity="0"
+									Text="Xg" />
+								<Rectangle
+									x:Name="PlaceholderRect"
+									Fill="{ThemeResource ListViewItemPlaceholderBackgroundThemeBrush}"
+									Visibility="Collapsed" />
+								<Rectangle
+									x:Name="MultiArrangeOverlayBackground"
+									Grid.ColumnSpan="2"
+									Fill="{ThemeResource ListViewItemDragBackgroundThemeBrush}"
+									IsHitTestVisible="False"
+									Opacity="0" />
+								<Border
+									x:Name="MultiSelectSquare"
+									Width="20"
+									Height="20"
+									Margin="12,0,0,0"
+									HorizontalAlignment="Left"
+									VerticalAlignment="Center"
+									BorderBrush="{ThemeResource SystemControlForegroundBaseMediumHighBrush}"
+									BorderThickness="2"
+									Visibility="Collapsed">
+									<Border.Clip>
+										<RectangleGeometry Rect="0,0,20,20">
+											<win:RectangleGeometry.Transform>
+												<TranslateTransform x:Name="MultiSelectClipTransform" />
+											</win:RectangleGeometry.Transform>
+										</RectangleGeometry>
+									</Border.Clip>
+									<Border.RenderTransform>
+										<TranslateTransform x:Name="MultiSelectCheckBoxTransform" />
+									</Border.RenderTransform>
+									<FontIcon
+										x:Name="MultiSelectCheck"
+										win:Glyph="&#xE73E;"
+										xamarin:Glyph="&#xE081;"
+										FontFamily="{ThemeResource SymbolThemeFontFamily}"
+										FontSize="16"
+										Foreground="{ThemeResource SystemControlForegroundBaseMediumHighBrush}"
+										Opacity="0"
+										Visibility="Collapsed" />
+								</Border>
+								<TextBlock
+									x:Name="MultiArrangeOverlayText"
+									Grid.ColumnSpan="2"
+									Margin="18,9,18,0"
+									AutomationProperties.AccessibilityView="Raw"
+									FontFamily="{ThemeResource ContentControlThemeFontFamily}"
+									FontSize="26.667"
+									Foreground="{ThemeResource ListViewItemDragForegroundThemeBrush}"
+									IsHitTestVisible="False"
+									Opacity="0"
+									Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.DragItemsCount}"
+									TextTrimming="WordEllipsis"
+									TextWrapping="Wrap" />
+								<VisualStateManager.VisualStateGroups>
```

Comment:
we can trim the fat here a little by removing the unused multi-select, drag-and-drop, reorder states

Reply:
Removed